bubo/pindora_api.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import json
+from bubo.config import Config
+def get_headers():
+    headers = {
+        'Pindora-Api-Key': f'{Config.pindora_token}',

```

Comment:
Pretty sure this wont work. You need to initialize `Config`, see https://github.com/elokapina/middleman/blob/a0e20a22596651cd6f5841ee36f10f002bbf527a/middleman-bot#L20-L20

The other option is to pass an initialized instance in when calling functions here.